relation_engine_server/utils/arango_client.py

```diff
@@ -192,8 +358,47 @@ def create_view(name, config):
             raise ArangoServerError(resp.text)
 
 
-def get_analyzers(name):
-    pass
+def get_all_analyzers():
+    """
+    Fetch all existing analyzers from server
+    Resp to GET /_api/analyzer is
+    {
+        "error" : false,
+        "code" : 200,
+        "result" : [
+            {
+                "name" : "text_pt",
+                "type" : "text",
+                "properties" : {
+                    "locale" : "pt.utf-8",
+                    "case" : "lower",
+                    "stopwords" : [ ],
+                    "accent" : false,
+                    "stemming" : true
+                },
+                "features" : [
+                    "frequency",
+                    "norm",
+                    "position"
+                ]
+            },
+            ...
+        ]
+    }
+
+    Returns
+    [
+        { ... }
+    ]
+    """
+    resp = requests.get(
+        url=_CONF["api_url"] + "/analyzer",
+        auth=(_CONF["db_user"], _CONF["db_pass"]),
+    )
```

Comment:
can this use the `adb_request` function?

Reply:
Yes I missed this one